src/modelmanager.cpp

```diff
@@ -900,8 +898,10 @@ Status ModelManager::loadConfig(const std::string& jsonFilename) {
             std::this_thread::sleep_for(std::chrono::milliseconds(WRONG_CONFIG_FILE_RETRY_DELAY_MS));
             continue;
         }
-        rapidjson::IStreamWrapper isw(ifs);
-        rapidjson::ParseResult parseResult = configJson.ParseStream(isw);
+        std::stringstream config;
+        config << ifs.rdbuf();
+        md5 = FileSystem::getStringMD5(config.str());
```

Comment:
Every time you call stringstream::str() you create string copy. Create std::string once.